feat(retro-achievements): init

Mahonzu opened the pull request:
## 🎉 Theme for RetroAchievements 🎉

<!--
You should give a short description of the website that you have themed.
E.g. YouTube is a video sharing platform that allows users to upload, view, and share videos.

You should also attach some screenshots of the themed website, show it off!
-->

Retro Achievements is a website dedicated to adding and getting achievements in retro games. 
<img width="1851" height="1066" alt="catwalk" src="https://github.com/user-attachments/assets/85f24a16-dd0d-4fe8-976b-b552545773fa" />
<img width="1851" height="1066" alt="5" src="https://github.com/user-attachments/assets/f8e3125f-d696-4a93-b46b-22a37cbd5350" />
<img width="1851" height="1066" alt="4" src="https://github.com/user-attachments/assets/9c350e9d-ff01-4407-a6a8-6f490dee5e5d" />
<img width="1851" height="1066" alt="3" src="https://github.com/user-attachments/assets/10cdd53f-1199-4cfa-9dbd-d3cae4f2a6f8" />
<img width="1851" height="1066" alt="1" src="https://github.com/user-attachments/assets/cc844d19-8170-45b0-b708-353d85718b83" />
<img width="1851" height="1066" alt="2" src="https://github.com/user-attachments/assets/76862c5c-eb68-4bbf-8618-fc75c9acea9f" />

## 💬 Additional Comments 💬

<!--
Include any difficulties you had theming this port, or any general comments that would be useful for the reviewer to know.
Feel free to leave this section empty if you don't have anything more to say.
-->
First time contributing to a project!!
I'm ready to hear constructive criticism, fix all issues and maintain the userstyle as long as I can.

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [submission guidelines](https://userstyles.catppuccin.com/contributing/creating-userstyles/).
- [X] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [X] I have ensured that the new directory is in **lower-kebab-case**.
  - [X] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [X] I have made sure to update the [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml) file with information about the new userstyle.
- [X] I have included the following files:
  - [X] `catppuccin.user.less` - all the CSS for the userstyle, based on the template.

Mahonzu committed:
feat: add RetroAchievements style

Mahonzu committed:
Fixes

- Fixed Softcore progressbar
 - Fixed Achievement page having wrong background color

Mahonzu committed:
Update userstyles.yml

WalkQuackBack commented on styles/retro-achievements/catppuccin.user.less:
```diff
@@ -0,0 +1,184 @@
+            background-color: var(--embed-highlight-color);
+        }
+        .border-neutral-200 {
+            border-color: var();
```
Should this be set to something?

Mahonzu replied:
I made a mistake while moving from CSS to LESS, thank you for pointing this out

AlwaysNur commented on styles/retro-achievements/catppuccin.user.less:
```diff
@@ -0,0 +1,184 @@
+@name RetroAchievements Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/retro-achievements
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/retro-achievements
+@version 1.0
```
this updates automatically so we leave it at that

AlwaysNur replied:
Oh, sorry mb I read the diff wrong :P 
I read it as:
```diff
+ @version 1.0
- @version 2000.01.01
```

Mahonzu committed:
Fixes

- Fixed version number
- Fixed theme selector
- Fixed empty class
- Fixed typo in comment
- Fixed case in userstyles.yml
- Removed fields from suerstyles.yml

Mahonzu committed:
Re-add fields to userstyles.yml

I read the diff wrong way, my bad

Mahonzu commented on scripts/userstyles.yml:
```diff
@@ -785,6 +786,13 @@ userstyles:
+    categories: [entertainment, games]
+    color: red
+    current-maintainers: [*Mahonzu]
+    past-maintainers: []
```
Noted

Mahonzu committed:
feat(retro-achievements): Add support for event pages

- Added support for Event pages
- Sorted styles by pages
- Added coments to make finding pages easier

Mahonzu committed:
fix(retro-achievements): Remove "games" tag

Mahonzu committed:
fix(retro-achievements): Fix profile colors

Before this profile page was using default yellow colors for progress
and masteries

WalkQuackBack commented on styles/retro-achievements/catppuccin.user.less:
```diff
@@ -0,0 +1,345 @@
+        }
+        /* Badge achievement count card */
+        .bg-white\/5 {
+            backgroudn-color: @overlay0
```
```suggestion
            background-color: @overlay0
```

Mahonzu committed:
fix(retro-achievements): Fix duplicate selector

Mahonzu committed:
fix(retro-achievements): Fix typo

Mahonzu committed:
fix(retro-achievements): Fix online history graph sometimes ignoring accent color

Mahonzu committed:
fix(retro-achievements): Fix Achievement Distribution graph ignoring accent color

Mahonzu committed:
fix(retro-achievements): Fix subset toggle buttons using wrong color

Mahonzu committed:
fix(retro-achievements): Fix searchbar colors
